controllers/articles-controller.js

```diff
@@ -28,3 +29,24 @@ exports.getArticles = (request, response, next) => {
       }
     });
 };
+
+exports.patchArticleVotes = (request, response, next) => {
+  const { article_id } = request.params;
+  const { inc_votes } = request.body;
+
+  if (typeof inc_votes !== "number") {
+    return response.status(400).json({ msg: "Bad Request" });
+  } else if (inc_votes === undefined) {
+    return response.status(400).json({ msg: "Bad Request" });
+  }
```

Comment:
if inc_votes is undefined, psql will handle it with code 23502 . psql will also handle wrong datatype. It's better to let psql handle it for you rather than creating new customs errors

Reply:
Just saw these comments, will make these changes, thanks!